config/generic-mks-sgenl-v2.cfg

```diff
@@ -0,0 +1,216 @@
+[printer]
+kinematics: cartesian
+max_velocity: 500
+max_accel: 3000
+max_z_velocity: 25
+max_z_accel: 30
+
+
+[mcu]
+    serial:/dev/serial/by-id/usb-Klipper_lpc1769_0D80000EA29869AFD0E63F5EC72000F5-if00
+
+##--------------------------------------------------------------------
+#  Stepper Settings
+##--------------------------------------------------------------------
+
+[stepper_x]
+step_pin: P2.2
+dir_pin: P2.3
+enable_pin: !P2.1
+rotation_distance: 40
+microsteps: 16
+full_steps_per_rotation:200  #set to 400 for 0.9 degree stepper
+endstop_pin: ^P1.29
+position_endstop: -10
+position_min: -10
+position_max: 263
+homing_speed: 200
+second_homing_speed: 20
+homing_retract_dist: 5
+
+[tmc2209 stepper_x]
+uart_pin: P1.1
+run_current: 0.7
+sense_resistor: 0.110
+stealthchop_threshold: 0
+##--------------------------------------------------------------------
+## Y Stepper
+##--------------------------------------------------------------------
+[stepper_y]
+step_pin: P0.19
+dir_pin: P0.20
+enable_pin: !P2.8
+rotation_distance: 40
+microsteps: 16
+full_steps_per_rotation:200
+endstop_pin: ^P1.27
+position_endstop: -20
+position_min: -20
+position_max: 220
+homing_speed: 200
+second_homing_speed: 20
+homing_retract_dist: 5
+
+## Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_y]
+uart_pin: P1.8
+run_current: 0.7
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##--------------------------------------------------------------------
+#Z Stepper Settings
+##--------------------------------------------------------------------
+
+## Z0 Stepper
+[stepper_z]
+step_pin: P0.22
+dir_pin: P2.11
+enable_pin: !P0.21
+rotation_distance: 8
+full_steps_per_rotation:400
+microsteps: 16
+endstop_pin: ^!P1.28
+position_endstop: 0
+position_min: -1.3
+position_max: 200
+homing_positive_dir: false
+homing_speed: 6
+second_homing_speed: 4
+homing_retract_dist: 3
+
+## Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z]
+uart_pin: P1.10
+run_current: 0.7
+sense_resistor: 0.110
+stealthchop_threshold: 0
```

Comment:
Not needed. Default anyway.

Reply:
are you referring to "stealthchop_threshold" ?

Reply:
Yes